src/libs/ReportActionsUtils.ts

```diff
@@ -117,6 +130,17 @@ function isWhisperAction(reportAction: OnyxEntry<ReportAction>): boolean {
     return (reportAction?.whisperedToAccountIDs ?? []).length > 0;
 }
 
+function isWhisperActionTargetedToOthers(reportAction: OnyxEntry<ReportAction>): boolean {
```

Comment:
NAB -  So this function checks if the whisper is targeting someone other than you, right? I feel like this function title could be clarified, or maybe just a function doc explaining what this does could make it super clear. Thoughts?

Reply:
added a doc